rbx_types/src/attributes/writer.rs

```diff
@@ -0,0 +1,121 @@
+use super::*;
+use crate::{basic_types::*, variant::Variant};
+
+fn write_f32(bytes: &mut Vec<u8>, n: f32) {
+    bytes.append(&mut n.to_le_bytes().to_vec());
```

Comment:
We should use `write_all` here instead like we do in rbx_binary. It replaces this with `bytes.write_all(&n.to_le_bytes()).unwrap()`. `write_all` shouldn't ever fail in this case (it will only fail if the Vec is too large afaik) so `unwrap` should be safe to use.

See `write_u32` as well.